google/cloud/internal/rest_response.cc

```diff
@@ -136,6 +136,17 @@ StatusCode MapHttpCodeToStatus(std::int32_t code) {
   return StatusCode::kUnknown;
 }
 
+bool IsHttpSuccess(RestResponse const& response) {
+  static_assert(HttpStatusCode::kMinSuccess < HttpStatusCode::kMinNotSuccess,
+                "Invalid HTTP code success range");
+  return response.StatusCode() <= HttpStatusCode::kMinNotSuccess &&
```

Comment:
It feels like either `kMinNotSuccess` is lying, or that should be `<'.  That is, "== kMinNotSuccess" shouldn't be a success.  (??)

Reply:
Good catch, fixed.